File: lovely/dump/conf.lua
LOVELY_INTEGRITY = '1e2499c2cbb65ceeb296ebe91a714618872b984ad4b5e7e9df6735355bc2371f'

_RELEASE_MODE = true
_DEMO = false

function love.conf(t)
	t.console = not _RELEASE_MODE
	t.title = 'Balatro'
	t.window.width = 0
    t.window.height = 0
	t.window.minwidth = 100
	t.window.minheight = 100
end 

File: lovely-ts/dump/conf.ts
///<reference types="lua-types/jit"/>
///<reference types="love-typescript-definitions"/>
///<reference path="./globals.ts"/>

var _RELEASE_MODE = true;
var _DEMO = false;

love.conf = function (t: import("love").Config) {
  t.console = !_RELEASE_MODE;
  t.title = "Balatro";
  t.window.title = "Balatro";
  t.window.width = 0;
  t.window.height = 0;
  t.window.minwidth = 100;
  t.window.minheight = 100;
};
